heartbeat/aws-vpc-move-ip

```diff
@@ -154,6 +156,15 @@ Enable enhanced monitoring using AWS API calls to check route table entry
 <shortdesc lang="en">Enhanced Monitoring</shortdesc>
 <content type="boolean" default="${OCF_RESKEY_monapi_default}" />
 </parameter>
+
+<parameter name="lookup_type" required="0">
+<longdesc lang="en">
+Name of resource type to lookup in route table. In shared subnet setups use NetworkInterfaceId.
```

Comment:
Nice, this seems cleaner :)

Just for clarity I'm in favor of telling the user what the allowed values for this are. See the `Filesystem` RA's `run_fsck` parameter for an example: https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/Filesystem#L192-L202

See also `vg_access_mode` and `activation_mode` params of the `LVM-activate` RA: https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/LVM-activate#L149-L168

Of course, in this case the user could deduce the two available options from the "use NetworkInterfaceId" statement together with the default value. But IMO best to be clear that those are the full set of supported values. Also, the default value isn't printed by `pcs resource describe <RA>`. (I should file an RFE for that today.)